### ISSUE TITLE
Message as auxiliary file

### ISSUE THREAD

**Issue by smnbbrv** [appleboy/drone-telegram#71]
Hi @appleboy 

the plugin is awesome, I really like it!

The only thing that bothers me (a little) is that I cannot configure the message in a side file.

It would be incredible if one is able to do the following:

Let's say: in `.drone-telegram-message.md`

```markdown
{{#success build.status}}
... here goes md content
{{else}}
... here goes md content
{{/success}}
```

Pros:
- file is supported by IDE by simply following its extension / file type
- no need to encode special yaml characters such as `#`, `:` etc.
- more readable .drone.yml

And then the config would look as

```yml
    name: telegram
    depends_on: [dockerize]
    image: appleboy/drone-telegram
    when:
      status: [success, failure]
    settings:
      token:
        from_secret: telegram_token
      to:
        from_secret: telegram_to
      format: markdown
      messageFile: .drone-telegram-message.md
```

What do you think?

**Comment by smnbbrv** [comment]
This could potentially fix issues like this one #67

**Comment by appleboy** [comment]
@smnbbrv OK. I will implement this feature.
